src/bundlers/node/nodeJsBundler.ts

```diff
@@ -430,6 +436,12 @@ export class NodeJsBundler implements BundlerInterface {
         debugLogger.debug(
             `[NodeJSBundler] Copy non js files and node_modules for file ${input.path}.`,
         );
+
+        const isDeployedToCluster = input.projectConfiguration.cloudProvider === "cluster";
+        const nodeVersion =
+            input.projectConfiguration.options?.nodeRuntime === "nodejs18.x" ? "18" : "16";
```

Comment:
Two different comments:
- please do not use hardcoded values. You can use these global variables [1].
- the default value for the node runtime is `nodejs20.x` as changed in this PR [2].

[1] https://github.com/Genez-io/genezio/blob/main/src/models/nodeRuntime.ts#L2
[2] https://github.com/Genez-io/genezio/pull/852/files#diff-ea6d78292f2749bc22525a587e7283bb1d9984569a9d861999509e8c84abe6ee